content/docs/reactivesearch/searchbase/overview/apireference.md

```diff
@@ -357,6 +360,7 @@ These properties are automatically calculated or managed by the `Searchbase` cla
     -   **`hidden`**: `number` Total number of hidden results found
     -   **`promoted`**: `number` Total number of promoted results found
     -   **`promotedData`**: `Array<Object>` An array of promoted results obtained from the applied query.
+    -   **`customData`**: `Object` An object of custom data obtained from the `reactivesearch-v3` API.
```

Comment:
Whenever referencing the ReactiveSearch v3 API, link to the doc on it. This doc is different from the REST API reference doc. @bietkul We need to create this doc.

Reply:
@siddharthlatest have added TODOs, will link them once the docs for the same are ready.

Reply:
@ShahAnuj2610 The docs are ready and live. You can now make the above change.